### PR TITLE
update the featrueGate PruneManagedFields and PruneLastAppliedConfiguration to beta

### PR DESCRIPTION
**Pull request by calvin0327**
fixed: https://github.com/clusterpedia-io/clusterpedia/issues/167

update the featrueGate PruneManagedFields and PruneLastAppliedConfiguration to beta

/kind feature
/cc @Iceber

### CODE REVIEW

File: charts/templates/clustersynchro-manager-deployment.yaml [hunk 1]
```diff
@@ -37,7 +37,6 @@ spec:
         - /usr/local/bin/clustersynchro-manager
         - --storage-config=/etc/clusterpedia/storage/internalstorage-config.yaml
         - --leader-elect-resource-namespace={{ .Release.Namespace }}
-        - --feature-gates=PruneManagedFields=true,PruneLastAppliedConfiguration=true
```
Comment (Iceber) [inline charts/templates/clustersynchro-manager-deployment.yaml:40]: Would it be better to set it explicitly for the feature gate we define and use?

Comment (calvin0327) [reply charts/templates/clustersynchro-manager-deployment.yaml:40]: The feature gate is beta at current stage, we can set it explicitly.  if the feature gate update to GA, we don't have to specify it, do you think?

Comment (Iceber) [reply charts/templates/clustersynchro-manager-deployment.yaml:40]: > The feature gate is beta at current stage, we can set it explicitly. if the feature gate update to GA, we don't have to specify it

I think it's fine

**Review by Iceber — APPROVED** [review]
LGTM

**Pull request merged**